api/models.py

```diff
@@ -64,3 +64,17 @@ class Task (models.Model):
 	deadline = models.DateTimeField(blank=True, null=True)
 	date_created = models.DateTimeField(auto_now_add=True)
 	date_updated = models.DateTimeField(auto_now=True)
+
+class Event(models.Model):
+
+	title = models.CharField(max_length=255, blank=False, null=False)
+	start_date = models.DateTimeField(blank=False, null=False)
+	end_date = models.DateTimeField()
+	start_time = models.DateTimeField(blank=False, null=False)
```

Comment:
That wasn't really the suggestion. A DateTimeField includes both the date and time, so it doesn't make sense to have two such fields.